### PR TITLE
[IMP] Productivity: Outlook Plugin edits

### PR DESCRIPTION
**Pull request by tiku-odoo**
Tasks: 
https://www.odoo.com/web#id=3594588&cids=3&model=project.task&view_type=form
https://www.odoo.com/web#id=3574085&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@LNAodoo When you have a moment can you take a look at this error that I'm receiving when testing:

**Comment by tiku-odoo** [comment]
@StraubCreative 

This PR is ready for your review. This is a quick change of the URL step to a file upload for the XML upload. 

I've also reworded and got rid of the lists. 

This is labeled priority from LNA

Thanks

File: content/applications/productivity/mail_plugins/outlook.rst [hunk 1]
```diff
@@ -118,20 +118,30 @@ Add a shortcut to the plugin
 By default, the Outlook Plugin can be opened from the *More actions* menu. However, to save
 time, it's possible to add it next to the other default actions.
```
Comment (StraubCreative) [inline content/applications/productivity/mail_plugins/outlook.rst:119]: early line break here

File: content/applications/productivity/mail_plugins/outlook.rst [hunk 2]
```diff
@@ -12,103 +15,100 @@ The Outlook :doc:`Mail Plugin <../mail_plugins>` needs to be configured both on
...
 
-      For Google Chrome, you can do so by following the guide at:
-      `https://support.google.com/chrome/answer/95647 <https://support.google.com/chrome/answer/95647#:~:text=Allow%20or%20block%20cookies%20for%20a%20specific%20site>`_
-      and adding `download.odoo.com` to the list of `Sites that can always use cookies`.
+   To fix this issue, configure the browser to always allow cookies on Odoo's plugin page.
 
-      Once done, the Outlook panel needs to be opened again.
+   For Google Chrome, change the browser cookie settings by following the guide at:
+   `https://support.google.com/chrome/answer/95647
+   <https://support.google.com/chrome/answer/95647#:~:text=Allow%20or%20block%20cookies%20for%20a%20specific%20site>`_
```
Comment (StraubCreative) [inline content/applications/productivity/mail_plugins/outlook.rst:89]: don't need the hyperlink anchor here
```suggestion
   <https://support.google.com/chrome/answer/95647>`_
```

**Review by StraubCreative — APPROVED** [review]
Hi @tiku-odoo 

Approving with comments
- line 89 is too long
- line 118 was broken too early.
- general comment: missing written specificity in a number of sentences and relying too much on screenshots
- grammar: we're missing some conjunctions in some spots so sentences that seem complete are actually fragments.

@robodoo delegate=tiku-odoo

**Comment by tiku-odoo** [comment]
@robodoo r+

**Pull request closed without merging**